qubes/api/__init__.py

```diff
@@ -94,6 +97,33 @@ def apply_filters(iterable, filters):
         iterable = filter(selector, iterable)
     return iterable
 
+# pylint: disable=line-too-long
+_uuid_regex = re.compile(rb"\A@uuid:[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}\Z")
+_qube_name_regex = re.compile(rb"\A[A-Za-z][A-Za-z0-9_.-]*\Z")
```

Comment:
Do not duplicate name validation code, there is `qubes.vm.validate_name` already  (you can pass None for holder and prop args).